pkg/render/logstorage.go

```diff
@@ -2122,17 +2132,52 @@ func (m managedClusterLogStorage) linseedExternalRoleAndBinding() (*rbacv1.Clust
 		},
 	}
 
-	// Bind the permission to the tigera-fluentd namespace. Other controllers may also bind
+	// These permissions are necessary so that we can fetch the operator namespace of the managed cluster from the
+	// management cluster so that we're copying secrets into the right place in a multi-tenant environment.
+	configMapsRole := &rbacv1.ClusterRole{
```

Comment:
Just to be clear - this configmap role is only needed in managed clusters, right? It isn't needed in a management cluster?

Reply:
That's right. We use the managed cluster's k8s client to call `FetchOperatorNamespace` which is what necessitates this permission.